### PR TITLE
Content build fails for long Install Content step - reconnecting client on each instance

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/30811

## Description
Fixed an issue where build failed on a long running (more than an hour) because demisto api token was not update. Addded
a function that reconnect to client on each instance. 

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Tests/configure_and_test_integration_instances.py [hunk 1]
```diff
@@ -1165,10 +1170,11 @@ def instance_testing(build: Build, all_module_instances, pre_update):
     else:
         logging.info(f'No integrations to configure for the chosen tests. ({update_status}-update)')
 
-    testing_client = build.servers[0].client
+    #testing_client = build.servers[0].client
     for instance in all_module_instances:
         integration_of_instance = instance.get('brand', '')
         instance_name = instance.get('name', '')
+        testing_client = build.servers[0].reconnect_client
```
Comment (yaakovi) [inline Tests/configure_and_test_integration_instances.py:1177]:
```suggestion
        testing_client = build.servers[0].reconnect_client()
```

**Comment by abaumgarten** [comment]
I did a minor (added new line) in DeleteContext script in order to test many files in Content. I will delete this change after review.

File: Packs/CommonScripts/Scripts/script-DeleteConext.yml [hunk 2]
```diff
@@ -16,6 +16,7 @@ script: >
 
   var shouldDeleteAll = (args.all === 'yes');
 
+
```
Comment (DeanArbel) [inline Packs/CommonScripts/Scripts/script-DeleteConext.yml:19]: Don't forget to remove this
```suggestion
```

Comment (abaumgarten) [reply Packs/CommonScripts/Scripts/script-DeleteConext.yml:19]: yes, sure. I am waiting to the build to be successfully done and the I'll remove that line.

**Review by guykeller — APPROVED** [review]

**Comment by guykeller** [comment]
@abaumgarten Let's make the build pass
@yaakovi Can you please double check me here? I don't see a reason why reconnecting the client for each instance would cause much harm, but I might be missing something here

**Comment by abaumgarten** [comment]
no documentation was added, labelled as docs approved

**Comment by abaumgarten** [comment]
Looks like last failure was due to build timed out.
server 6.0 : https://app.circleci.com/pipelines/github/demisto/content/47152/workflows/0bf2f380-8799-4bda-a485-5df1bf4dcb21/jobs/201992
server master : https://app.circleci.com/pipelines/github/demisto/content/47152/workflows/0bf2f380-8799-4bda-a485-5df1bf4dcb21/jobs/201993

However, it looks like the error that @DeanArbel attached does not happen anymore (https://app.circleci.com/pipelines/github/demisto/content/45592/workflows/56db856a-5b8c-4dfd-9ee4-d89a65e0f941/jobs/195720) .

File: Packs/CommonScripts/Scripts/script-Set.yml [hunk 3]
```diff
@@ -8,6 +8,7 @@ script: >-
   var value;
 
   try {
+
```
Comment (yaakovi) [inline Packs/CommonScripts/Scripts/script-Set.yml:11]: don't forget to revert

**Review by yaakovi — APPROVED** [review]

File: Packs/CommonScripts/Scripts/script-Set.yml (hunk 3 above)
Comment (abaumgarten) [reply Packs/CommonScripts/Scripts/script-Set.yml:11]: doing that right now.

**Pull request merged**